README.md

```diff
@@ -17,16 +17,12 @@ Group todo list app created as part of the Google Developer Scholarship 2017
   npm install
   ```
   in the project root.
-* To build the code run
+* To build the code and run the webserver on port 3000
   ```bash
-  npm run build
+  npm run start
   ```
```

Comment:
I have created a new command "npm run dev" for development purpose so it will not modify an existing command 
https://github.com/thornecc/grouptodo/commit/35bdecd5d7e9337d982890156aba6486c86957c4